frameworks/elastic/tests/test_tls.py

```diff
@@ -91,3 +115,10 @@ def test_crud_over_tls(elastic_service_tls):
 
     assert document
     assert document['_source']['name'] == 'Loren'
+
+
+@pytest.mark.tls
+@pytest.mark.sanity
+@pytest.mark.dcos_min_version('1.10')
```

Comment:
Since all tests are `dcos_min_version('1.10')` we could add this to the module-level `pytestmark`.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2349/commits/a6b8f0c1b61f37bae4e7dc071a0aff30a6cacbc6